.travis.yml

```diff
@@ -1,31 +1,35 @@
 language: python
-branches:
-    only:
-        - master
-        - dogfood
-services:
-  - postgres
 python:
   - "2.7"
+
+virtualenv:
+  system_site_packages: true
+
+sudo: required
+dist: trusty
+
+cache:
+  - pip
+
+addons:
+  postgresql: "9.3"
```

Comment:
Does 9.3 vs 9.4 or 9.5 matter much?

Are we specifying it here because we have to specify some version?